app/scripts/metamask-controller.js

```diff
@@ -2054,8 +2054,7 @@ export default class MetamaskController extends EventEmitter {
       decodingApiUrl: process.env.DECODING_API_URL,
       isDecodeSignatureRequestEnabled: () =>
         this.preferencesController.state.useExternalServices === true &&
-        this.preferencesController.state.useTransactionSimulations &&
-        process.env.ENABLE_SIGNATURE_DECODING === true,
```

Comment:
Just to clarify, is this PR _also_ enabling signature decoding? Where previously it was disabled by default? It's very misleading for a PR titled "chore: add e2e test" to do that as well.

Reply:
Better to do this in a separate PR. Or at the very least, update the PR title to reflect the functional change here.

Reply:
Yep it is, I updated PR title to reflect that.

Decoding signature was disabled previously as it was under development.